src/App.vue

```diff
@@ -223,6 +234,10 @@ export default {
 			}
 		},
 
+		openHelp() {
+			this.helpVisible = !this.helpVisible
+			this.helpVisible = true
```

Comment:
Shouldn't this be sufficient?
```suggestion
			this.helpVisible = true
```

Reply:
Sadly no. If i only set it to true, we can only open the help once. If the user then closes the help, he or she has to reload the page to make it work. 

I guess vue (or more likely me) does not propagate the helpVisible state from the help component back up.